### PR TITLE
1 yum

### PR DESCRIPTION
**Pull request by jvonau**

### CODE REVIEW

File: install-init.yml [hunk 1]
```diff
@@ -29,4 +27,4 @@
     when: config_vars.stat.exists == false
 
   - name: Install Initionalization Complete
-    command: echo Install Initionalization Complete
\ No newline at end of file
+    command: echo Install Initionalization Complete
```
Comment (davelab6) [inline install-init.yml]: `Initionalization` seems odd English.`Initialization` is more typical.

**Comment by davelab6** [comment]
At a high level what does all this do?

**Comment by jvonau** [comment]
@davelab6 This routine lays the foundation for the XSCE install, does a bunch of prep, installs rpms, pips in order to setup access to the cmd_srv web frontend at <url>/admin to allow further services to be enabled and content added.

**Comment by jvonau** [comment]
superseded by #634

**Comment by jvonau** [comment]
clean version at #675

**Pull request closed without merging**